core/dbt/cli/requires.py

```diff
@@ -42,6 +42,7 @@ def wrapper(*args, **kwargs):
 
         # Deprecation warnings
         [dep_fn() for dep_fn in flags.deprecated_env_var_warnings]
```

Comment:
Nit: should we combing this line and the next line into the a single function that's called here? I guess we can't do it in flags init because logger is not ready when we construct flags?